src/dotnet/commands/dotnet-new/CSharp_Web/$projectName$.csproj

```diff
@@ -1,4 +1,4 @@
-<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project ToolsVersion="15.0">
   <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" />
 
   <PropertyGroup>
```

Comment:
We should remove the Exlcude=$(GlobalExclude) from the Compile and EmbeddedResource includes.  The SDK already has a Remove entry for the excludes and so we don't need it in the project file.